stylix/palette.nix

```
@@ -74,6 +80,26 @@ in {
         doesnt fix your monitor perfectly
       '';
     };
+    imageEditor = {
+      enable = lib.mkOption {
+        type = lib.types.bool;
+        default = false;
+        description = ''
+          Update Wallpaper by applying a lut filter to the image
+        '';
+      };
+      method = lib.mkOption {
+        type = with lib.types; functionTo (coercedTo package toString path);
+        default = config.lib.stylix.imageEditors.lutgen;
+        description = ''
+          A function to edit the image, takes one argument (the image)
+          and returns the resulting edited image
+        '';
+        example = ''
+          config.stylix.imageEditor.method = config.lib.stylix.imageEditors.lutgen;
+        '';
```

Directly refer to value:

```suggestion
        example = "config.lib.stylix.imageEditors.lutgen;";
```